dd-sdk-android/src/main/kotlin/com/datadog/android/core/configuration/Configuration.kt

```diff
@@ -347,6 +349,11 @@ internal constructor(
             return this
         }
 
+        fun setBatchSize(batchSize: BatchSize): Builder {
```

Comment:
This new public API needs doc comment 💬.

Reply:
Done